stride/chain.json

```diff
@@ -245,8 +245,8 @@
     {
       "kind": "Nodeist Explorer",
       "url": "https://exp.nodeist.net/Stride",
```

Comment:
Can we update this account page too, please? It also just needs the 'M-'

Reply:
oh sorry ok!

Reply:
Why close the PR? it just needed a simple change